### PR TITLE
Ticket646 hifi cryo mag gaussmeter opi

### PR DESCRIPTION
**Pull request by Mukz95**
### Description of work

*Add your own description here*
ISISComputingGroup/IBEX#646
### To test

*Which ticket does this PR fix?*

### Acceptance criteria

*List the acceptance criteria for the PR*

---

#### Code Review

- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/GUI-Coding-Conventions)? Is it well structured with small focussed classes/methods/functions?
- [ ] Have no new checkstyle warnings been introduced? Check via Jenkins
- [ ] Are there unit tests in place? Are the unit tests small and test the a class in isolation?
- [ ] Are there automated [system tests](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/System-Testing-with-RCPTT) in place? Do they test a minimal set of functionality and leave the gui as close as possible to its original state?
- [ ] Has the [manual system tests spreadsheet](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Manual-system-tests) been updated?
    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests
- [ ] Did any existing system test break as a result of the current changes? 
- [x] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev). If so, do they describe the changes appropriately?
- [x] If an OPI has been modified, does it conform to the [style guidelines](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/OPI-Creation)? There is a script called `check_opi_format.py` to help with this.

### Functional Tests

- [x] Do changes function as described? Add comments below that describe the tests performed.
- [x] How do the changes handle unexpected situations, e.g. bad input?
- [x] Has developer documentation been updated if required?
- [ ] Have any new plugins been added to the appropriate feature.xml? You can check if this is needed by validating plugins as per method at  https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Common-Eclipse-Issues

### Final Steps
- [x] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml [hunk 1]
```diff
@@ -1129,5 +1129,19 @@
 				</macros>
 			</value>
 		</entry>
+        <entry>
+			<key>Lakeshore 460</key>
+			<value>
+				<type>Gaussmeter</type>
+				<path>lakeshore460.opi</path>
```
[the remark this answers is not in the document]
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml]: Added category (Monitors)

[the remark this answers is not in the document]
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml]: changed to tabbed containers

[the remark this answers is not in the document]
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml]: Added icon

File: base/uk.ac.stfc.isis.ibex.opis/resources/Eurotherm.opi [hunk 2]
```diff
@@ -463,7 +463,7 @@
       <macros>
         <include_parent_macros>true</include_parent_macros>
       </macros>
-      <visible>false</visible>
+      <visible>true</visible>
```
Comment (davidkeymer) [inline base/uk.ac.stfc.isis.ibex.opis/resources/Eurotherm.opi]: Did you really mean to edit the Eurotherm OPI?

File: base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml [hunk 3]
```diff
@@ -1129,5 +1129,22 @@
 				</macros>
 			</value>
 		</entry>
+        <entry>
+			<key>Lakeshore 460</key>
+			<value>
+			    <categories>
+					<category>Monitors</category>
```
Comment (davidkeymer) [inline base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml]: Is this the most appropriate category for this device?  "Monitors" is used for neutron monitors in the beam.

File: base/uk.ac.stfc.isis.ibex.opis/resources/Eurotherm.opi (hunk 2 above)
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/Eurotherm.opi]: Removed from branch

File: base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml (hunk 3 above)
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml]: Changed to 'gaussmeter'

File: base/uk.ac.stfc.isis.ibex.opis/resources/Eurotherm.opi (hunk 2 above)
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/Eurotherm.opi]: - Fixed title casing

- Moved units selector to 'Device' tab instead of channel tabs

**Pull request merged**